asmcli/lib/util.sh

```diff
@@ -349,7 +349,8 @@ init() {
   KPT_URL="${KPT_URL}.git/asm@${KPT_BRANCH}"; readonly KPT_URL;
   ISTIO_FOLDER_NAME="istio-${RELEASE}"; readonly ISTIO_FOLDER_NAME;
   ISTIOCTL_REL_PATH="${ISTIO_FOLDER_NAME}/bin/istioctl"; readonly ISTIOCTL_REL_PATH;
-  BASE_REL_PATH="${ISTIO_FOLDER_NAME}/manifests/charts/base/files/gen-istio-cluster.yaml"; readonly BASE_REL_PATH;
+  BASE_REL_PATH="${_CI_BASE_REL_PATH:-${ISTIO_FOLDER_NAME}/manifests/charts/base/files/gen-istio-cluster.yaml}"
```

Comment:
the reason I did it the other way is in the tests we will get istioctl from `./out/linux_amd64/istioctl` and the gen-istio-cluster from `./manifests/charts/base/files/gen-istio-cluster.yaml`

Reply:
The CI variable for istioctl gets used somewhere else, won't this work if you set _CI_BASE_REL_PATH to `./manifests/charts/base/files/gen-istio-cluster.yaml`?

Reply:
Oh I read it wrong. LGTM